clientd3d/music.c

```diff
@@ -321,38 +393,63 @@ DWORD PlayMusicFile(HWND hWndNotify, const char *fname)
    DWORD dwReturn;
    MCI_PLAY_PARMS mciPlayParms;
    char temp[81];
-
-   if ((dwReturn = OpenMidiFile(fname)) != 0)
+	debug(( "Play music file %s. LatestMusic=%d. playing_music=%d, playing_midi=%d \n", fname, latest_music, playing_music, playing_midi ));
```

Comment:
split long line

Reply:
I've split this in the code.  Do you want a line break in the actual debug message as well?